documentation.rst

```diff
@@ -14,20 +14,14 @@ see :ref:`here <Nengo ecosystem>`.
 
 Jump to documentation for:
 
-.. |nengocore| image:: https://www.nengo.ai/design/_images/general-full-light.svg
-   :height: 50
-   :alt: Nengo core
-   :target: https://www.nengo.ai/nengo
+.. |nengocore| replace:: Nengo
+.. _nengocore: https://github.com/nengo
```

Comment:
We should have these link to the rendered documentation pages (e.g. `https://www.nengo.ai/nengo`) for projects that have them.

Reply:
oops, not sure how I didn't notice that!